assets/wizards/site-design/views/theme-settings/index.js

```diff
@@ -35,6 +35,7 @@ const ThemeSettings = props => {
 		newspack_image_credits_placeholder_url: imageCreditsPlaceholderUrl,
 		newspack_image_credits_class_name: imageCreditsClassName = '',
 		newspack_image_credits_prefix_label: imageCreditsPrefix = '',
+		newspack_image_credits_auto_populate: imageCreditsAutoPopulate = true,
```

Comment:
This should also default to false per this comment: https://github.com/Automattic/newspack-plugin/pull/2973#issuecomment-1978779652

Reply:
Updated in 91dd7bed